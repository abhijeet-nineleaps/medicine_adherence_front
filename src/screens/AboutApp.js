import {Text, View, Image} from 'react-native';
import React from 'react';
import styles from "./screenStyles/AboutAppStyles";

const AboutApp = () => {
  return (
    <View style={styles.container}>
      <View style={styles.imageContainer}>
        <Image
          testID='aboutImg'
          style={styles.image}
          source={require('../../assests/images/Medstick_1.png')}
        />
      </View>
      <View style={styles.textContainer}>
        <Text testID='aboutText' style={styles.text}>
          Medicine Adherence app which allows user to use medicine, reminder, caretaker, patient, report and more features and never skip their medications.
        </Text>
      </View>
    </View>
  );
};

export default AboutApp;