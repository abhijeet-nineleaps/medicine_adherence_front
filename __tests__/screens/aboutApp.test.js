import React from 'react';
import renderer from 'react-test-renderer';
import AboutApp from '../../src/screens/AboutApp';

describe('About App', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AboutApp/>);
    expect(tree).toMatchSnapshot();
  });
  it('checks for text', () => {
    const tree = renderer.create(<AboutApp />);
    const text = tree.root.findByProps({testID: 'aboutText'}).props;
    expect(text.children).toEqual('Medicine Adherence app which allows user to use medicine, reminder, caretaker, patient, report and more features and never skip their medications.')
  })
});