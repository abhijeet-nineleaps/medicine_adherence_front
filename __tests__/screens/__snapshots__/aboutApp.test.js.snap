// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About App renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assests/images/Medstick_1.png",
        }
      }
      style={
        Object {
          "height": 466.9,
          "width": 750,
        }
      }
      testID="aboutImg"
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "gray",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginTop": 30,
          "paddingLeft": 4,
        }
      }
      testID="aboutText"
    >
      Medicine Adherence app which allows user to use medicine, reminder, caretaker, patient, report and more features and never skip their medications.
    </Text>
  </View>
</View>
`;
